src/components/common/CertificateBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import { CertificateBannerProps } from '@/interfaces';
+import { FaTrophy, FaLock } from 'react-icons/fa';
+
+const CertificateBanner = ({ totalChapters, completedChapters }: CertificateBannerProps) => {
+  if (completedChapters < totalChapters) {
```

Comment:
We're making it as Component which means it can be used at multiple places, Don't use Chapters. Simply add props to Locked or Unlocked Banner.